src/main/java/dumshenko/daniil/WordUtils.java

```diff
@@ -0,0 +1,42 @@
+package dumshenko.daniil;
+
+import java.util.*;
+
+public class WordUtils {
+
+    public static String getShortestWord(List<String> words) {
+        return words.stream()
+                .min(Comparator.comparingInt(String::length))
+                .orElse("No words provided");
+    }
+
+    public static Set<String> getUniqueWords(String sentence) {
+        List<String> words = Arrays.asList(sentence.split("\\s+")); // regex пiдказав чат
```

Comment:
`\\s+` is indeed a good way to split by spaces using regex but simple space is enough
```suggestion
        List<String> words = Arrays.asList(sentence.split(" "));
```